src/components/Alert.js

```diff
@@ -195,6 +195,8 @@ export const Alert = () => {
                 hide();
               }}
               label={button2.label}
+              width={undefined}
+              variant={button2.variant}
```

Comment:
@injas-bigbinary _a:  I think the default variant for button2 should be text based on https://www.figma.com/file/GvxNW24y6lmKkrzdZCIz2e/Design-System?node-id=471%3A1418   
```suggestion
              variant={button2.variant || "text"}
```